firefox-ios/firefox-ios-tests/Tests/ClientTests/BrowserViewControllerTests.swift

```diff
@@ -65,4 +65,36 @@ class BrowserViewControllerTests: XCTestCase {
 
         wait(for: [expectation], timeout: 5.0)
     }
+
+    func testTitleChangeEnablesReaderMode() {
+        guard let mockTab = loadMockTabWithTitle(title: "Webview title"), let tabWebView = mockTab.webView else { return }
```

Comment:
seems this test is failing it bitrise

Reply:
It is passing the test when tested repeatedly.  Seems like asynchronous wait is failing in Bitrise. I will update the wait time to see if the test case passes.